SConstruct

```diff
@@ -1478,12 +1478,20 @@ if env["hdf_libdir"]:
 if env["hdf_support"] == "n":
     env["use_hdf5"] = False
 else:
-    env["use_hdf5"] = conf.CheckLib("hdf5", autoadd=False)
+    libhdf_exists = conf.CheckLib("hdf5", autoadd=False)
+    libhdf_serial_exists = conf.CheckLib("hdf5_serial", autoadd=False)
+    env["use_hdf5"] = False
+    if libhdf_exists:
+        env["use_hdf5"] = True
+        env["hdf5_lib"] = "hdf5"
+    elif libhdf_serial_exists:
+        env["use_hdf5"] = True
+        env["hdf5_lib"] = "hdf5_serial"
+        add_system_include(env, "/usr/include/hdf5/serial/")
```

Comment:
I think we should only try adding this include path if no value has been set for `env["hdf_include"]`.

Reply:
Done! Thanks!